core/scripts/local/CreateTokens.js

```diff
@@ -0,0 +1,44 @@
+/**
```

Comment:
this is a super useful script. we need to re-org the script directory as it's getting out of hand now. Can add this to the mono repo refactor where we can think about how we want to store and organize all these little helper scripts. Relates to @nicholaspai's #1530